components/treelist/accessibility/wai-aria-support.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Wai-Aria Support
+page_title: Telerik UI for Blazor TreeList Documentation | TreeList  Accessibility
```

Comment:
Probably an issue with the template used for generating this page. There are two spaces between the component name and the word "accessibility".

```suggestion
page_title: Telerik UI for Blazor TreeList Documentation | TreeList Accessibility
```